book/src/designs/0002-parallel-verification.md

```diff
@@ -189,36 +118,6 @@ Here is how the `CheckpointVerifier` implements each verification stage:
 * **Constraint Verification:**
   * `process_checkpoint_range`: makes sure that the blocks in the checkpoint
     range have an unbroken chain of previous block hashes.
-* **Contextual Verification:**
```

Comment:
Rather than deleting this information, I'd like us to talk about how we maintain the context required by non-checkpoint blocks, during checkpoint verification.

Reply:
This is now split out into #763 and #871.